daemon/dns/test/interface-order

```diff
@@ -0,0 +1,34 @@
+# interface-order(5)
```

Comment:
It would make sense to split the part `resolvconfIfacePrefix` which parses the content of the file and returns the appropriate prefix and then make higher level function which opens a file and does that. By doing this, the function can be tested with unit tests.

Reply:
ok, done